features/specs/meerderjarig-verklaarde-minderjarige.feature

```diff
@@ -0,0 +1,207 @@
+    - Een erkenner van het kind krijgt geen gezag wanneer de erkenning heeft plaatsgevonden vóór de meerderjarigheidsverklaring, omdat de moeder op dat moment niet bevoegd was
+    - Een erkenner van het kind krijgt van rechtswege gezag wanneer de erkenning heeft plaatsgevonden na de meerderjarigheidsverklaring
+
+  Regel: Als de meerderjarigheidsverklaring is geregistreerd in de BRP bij de minderjarige moeder, dan is er eenhoofdig ouderlijk gezag voor de moeder en verliezen de ouders van de moeder het gezag over de moeder
+
+    Voorbeeld: De meerderjarig verklaarde moeder heeft gezag over haar kind
+      Gegeven de 17 jaar geleden geboren persoon 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Zoe'
+      En 1 maand geleden is 'Zoe' meerderjarig verklaard
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Zoe'
+
+    Voorbeeld: De meerderjarig verklaarde heeft gezag over haar kind en haar ouders hebben geen gezag meer over haar
+      Gegeven de 17 jaar geleden geboren persoon 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Zoe'
+      En 1 maand geleden is 'Zoe' meerderjarig verklaard
+      Als 'gezag' wordt gevraagd van 'Zoe'
+      Dan heeft 'Zoe' de volgende gezagsrelaties
+      * het gezag over 'Bert' is eenhoofdig ouderlijk gezag met ouder 'Zoe'
+
+    Voorbeeld: De ouders van de meerderjarig verklaarde hebben geen gezag over de meerderjarigverklaarde
+      Gegeven de 17 jaar geleden geboren persoon 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Zoe'
+      En 1 maand geleden is 'Zoe' meerderjarig verklaard
+      Als 'gezag' wordt gevraagd van 'Gerda'
+      Dan heeft 'Gerda' geen gezag
+
+  Regel: Een gerechtelijke uitspraak gezag voor toewijzing van gezag aan een minderjarige moeder geeft de moeder alleen gezag in combinatie met een meerderjarigheidsverklaring
+
+    Voorbeeld: Eerst is er voogdij uitgesproken over het kind en daarna is de moeder meerderjarig verklaard en is zij in een uitspraak met het gezag belast
+      Gegeven de 17 jaar geleden geboren persoon 'Gerda'
+      En de 1 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 10 maanden geleden is in een gerechtelijke uitspraak het gezag toegewezen aan een derde
+      En 1 maand geleden is 'Gerda' meerderjarig verklaard
+      En 1 maand geleden is in een gerechtelijke uitspraak het gezag toegewezen aan 'Gerda'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Eerst is er voogdij uitgesproken over het kind en daarna is zij in een uitspraak met het gezag belast zonder haar eerst meerderjarig te verklaren
```

Comment:
dit mag niet kunnen. is de "Amsterdam hack". moet dan worden opgelost met controle in de BCM